app/src/main/assets/ime/text/characters/mod/jis.json

```
@@ -0,0 +1,27 @@
+{
+  "type": "characters/mod",
+  "name": "jis",
+  "label": "JIS",
+  "authors": [ "waelwindows" ],
+  "direction": "ltr",
+  "arrangement": [
+    [
+      { "code":    0, "type": "placeholder" },
+      { "code":   -5, "label": "delete", "type": "enter_editing" }
+    ],
+    [
+      { "code": -202, "label": "view_symbols", "type": "system_gui" },
+      { "$": "variation_selector",
+        "default":  { "code":   44, "label": ",", "groupId": 1 },
+        "email":    { "code":   64, "label": "@", "groupId": 1 },
+        "uri":      { "code":   47, "label": "/", "groupId": 1 }
+      },
+      { "code":  -210, "label": "language_switch", "type": "system_gui" },
+      { "code":  -213, "label": "switch_to_media_context", "type": "system_gui" },
+      { "code":    32, "label": "空白" },
+      { "code": -9710, "label": "かな" },
```

The switcher is a system button, but the type attribute is not defined, so it defaults to `characters`, which does not allow for showing icons.

```suggestion
      { "code": -9710, "label": "かな", "type": "system_gui" },
```